### ISSUE TITLE
`yargs().env()` always fails in deno runtime

### ISSUE THREAD

**Issue by reosablo** [yargs/yargs-parser#405]
This is a small reproducible code:

```ts
// example.ts
import yargs from "https://deno.land/x/yargs@v17.0.1-deno/deno.ts";
yargs().env().parse([]);
```

```sh
deno run example.ts              # uncaught error: permission denied
deno run --allow-env example.ts  # uncaught error: Object.keys(undefined)
```

Maybe, that's because `env()` in [`deno.ts:12:14`](/yargs/yargs-parser/blob/v20.2.4-deno/deno.ts#L12-L14) returns nothing and doesn't handle `PermissionDenied` errors like [this](https://github.com/yargs/yargs/blob/v17.0.1-deno/lib/platform-shims/deno.ts#L28-L39):

```ts
  env: () => {
    Deno.env.toObject()
  },
```

**Comment by MaikuMori** [comment]
Can I make PR to fix this?

**Comment by reosablo** [comment]
@MaikuMori 
I didn't follow this issue but it seems to have been fixed in #432.
